### ISSUE TITLE
dark texture output of MPS backend on Mac M1 Pro

### ISSUE THREAD

**Issue by sekkit** [Stability-AI/stable-fast-3d#37]
dark texture output of MPS backend on Mac M1 Pro

**Comment by jammm** [comment]
Right, the output seems slightly darker. I suspect it's due to the MPS backend being a bit flaky as it's relatively new compared to CUDA. This is why we released it as experimental because the results don't exactly match that of the CPU backend. If you want the expected results, use `SF3D_USE_CPU=1` to run on the CPU backend.
